### ISSUE TITLE
contains: use `Contains` instead of `strings.Contains`

### ISSUE THREAD

**Issue by nickajacks1** [Antonboom/testifylint#54]
Using `assert.True` on `strings.Contains` does not print out useful information about the arguments in the case of a failure.
`assert.Contains`, on the other hand, will report both values on failure.
```go
❌
     assert.True(t, strings.Contains(s, "abc123"))
     assert.True(t, strings.Contains(string(b), "abc123"))

✅
     assert.Contains(t, s, "abc123")
     assert.Contains(t, string(b), "abc123")
```

`assert.Contains` doesn't seem to work well with byte slices, so users would need to convert the bytes to strings.  Consequently, I'm not too sure about including bytes.Contains in the same lint rule.

**Comment by Antonboom** [comment]
Hi, @nickajacks1!

Thank you for proposal.

> assert.Contains doesn't seem to work well with byte slices

offtop: maybe to open issue/PR in `testify`?

**Comment by nickajacks1** [comment]
Good idea, I'll look into that and report back here.

**Comment by Antonboom** [comment]
https://github.com/stretchr/testify/pull/1526

**Comment by ccoVeille** [comment]
For records, I started working on the implementation of this feature request.

Stay tuned, and ping me if you see no progress in the next month.
